packages/cli/src/commands/init/__tests__/editTemplate.test.ts

```diff
@@ -200,6 +201,67 @@ describe('changePlaceholderInTemplate', () => {
   );
 });
 
+const samplePackageJson: string = `{
+  "name": "HelloWorld",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
+    "lint": "eslint .",
+    "start": "react-native start",
+    "test": "jest"
+  },
+  "dependencies": {
+    "react": "19.0.0-rc-fb9a90fa48-20240614",
+    "react-native": "1000.0.0"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.20.0",
+    "@babel/preset-env": "^7.20.0",
+    "@babel/runtime": "^7.20.0",
+    "@react-native/babel-preset": "0.75.0-main",
+    "@react-native/eslint-config": "0.75.0-main",
+    "@react-native/metro-config": "0.75.0-main",
+    "@react-native/typescript-config": "0.75.0-main",
+    "@types/react": "^18.2.6",
+    "@types/react-test-renderer": "^18.0.0",
+    "babel-jest": "^29.6.3",
+    "eslint": "^8.19.0",
+    "jest": "^29.6.3",
+    "prettier": "2.8.8",
+    "react-test-renderer": "19.0.0-rc-fb9a90fa48-20240614",
+    "typescript": "5.0.4"
+  },
+  "engines": {
+    "node": ">=18"
+  }
+}`;
+
+describe('updateDependencies', () => {
+  beforeEach(() => {
+    jest.spyOn(process, 'cwd').mockImplementation(() => testPath);
+    jest.spyOn(fs, 'writeFileSync');
+    jest.spyOn(fs, 'readFileSync').mockImplementation(() => samplePackageJson);
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('updates react-native', () => {
+    updateDependencies({
+      dependencies: {
+        'react-native': '0.75.0',
+      },
```

Comment:
Let's test also `devDependencies` here?

Reply:
I'll add something that'll covers what we use to do this.